api/controllers/TagController.js

```diff
@@ -1,19 +1,90 @@
+var fetchAndPresentFollowed = (communityId, userId) =>
```

Comment:
and owner_id vs user_id.
Wasn't sure about refactoring, as it's not logically necessary that they're the same, they happen to be the same structure, but I can imagine that changing.
But sounds like you are for refactoring, so I'll do it.